es-ds-components/components/es-skeleton-img.vue

```diff
@@ -0,0 +1,36 @@
+<script setup lang="ts">
+const props = defineProps({
+    aspect: {
+        type: String,
+        default: '16:9',
+    },
+    noAspect: {
+        type: Boolean,
+        default: false,
+    },
+});
+
+const aspects = props.aspect.split(':');
+const aspectRatio = computed(() => `${(parseFloat(aspects[1]) / parseFloat(aspects[0])) * 100}%`);
+</script>
+
+<template>
+    <es-skeleton
+        v-if="noAspect"
+        v-bind="$attrs"
+        class="es-skeleton-img" />
+    <div
+        v-else
+        class="d-flex">
+        <div
+            class="aspect-sizer flex-grow-1"
```

Comment:
I don't see this `aspect-sizer` class defined anywhere in the repo, even in es-ds-styles - do we need it?

Reply:
I kept those classes as a vague idea of identifying the functionality of different containers/ helper components from source code - but not something we generally do and not needed for sure, took them out

Reply:
Ah - maybe could use comments instead? Then it wouldn't have a byte increase consequence for users.

Reply:
Good call! Added a couple comments